### PR TITLE
Added dark theme to home,result

### PR DESCRIPTION
**Pull request by pranavbharadwaj007**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,4 +1,14 @@
 /* NavBar */
+
+@import url('https://fonts.googleapis.com/css2?family=Fredoka+One&display=swap');
```
Comment (niranjanhegde144) [inline src/App.css:3]: Keep the same font on all the pages.

File: src/App.css [hunk 2]
```diff
@@ -76,30 +83,111 @@
...
+  /* border: 2px solid rgb(186, 231, 248); */
+}
+
+.paras {
+  padding: 0 60px;
+}
+
+.text1 {
+  color: rgb(226, 244, 247);
+  font-family: 'Fredoka One', cursive;
```
Comment (niranjanhegde144) [inline src/App.css:126]: Font used here is different. Add the font to the root element.

File: src/App.css [hunk 3]
```diff
@@ -76,30 +83,111 @@
...
+@media only screen and (max-width: 960px) {
   .bannerImageNotes {
     width: 100%;
     height: auto;
   }
+  /* .darkMode {
+    background: #0e0f11;
+    color: aliceblue;
+    height: 100%;
+  } */
```
Comment (niranjanhegde144) [inline src/App.css:190]: If it's not used remove it.

File: src/components/blogs.js [hunk 4]
```diff
@@ -3,12 +3,12 @@ import React, {Component} from 'react';
 class Blogs extends Component {
     render() {
         return(
-            <div>
+            <div className="darkMode">
+            <div >
                 <img src="https://blog.focusinfotech.com/wp-content/uploads/2017/08/blog-banner.jpg" alt="Blogs" width="100%" height="100%" />
             </div>
```
Comment (niranjanhegde144) [inline src/components/blogs.js:9]: Indent this my one tab or spaces.

File: src/index.css [hunk 5]
```diff
@@ -1,13 +1,10 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif;
```
Comment (niranjanhegde144) [inline src/index.css:3]: Even here keep the same font

**Review by niranjanhegde144 — CHANGES_REQUESTED** [review]
Have a single font-family in all the places. If you are in certain places font has to be different then you can specify the same with the comment.

File: src/App.css (hunk 1 above)
Comment (pranavbharadwaj007) [reply src/App.css:3]: Even in navbar notes also @niranjanhegde144

File: src/App.js [hunk 6]
```diff
@@ -15,7 +15,7 @@ import { Nav, Navbar } from "react-bootstrap";
 function App() {
   return (
     <div>
-      <Navbar bg="light" collapseOnSelect={true} varient="light" expand="lg">
+      <Navbar bg="dark" collapseOnSelect={true} varient="dark" expand="lg">
         <Navbar.Brand as={Link} to="/">
           <img
             src={process.env.PUBLIC_URL + "/favicon.ico"}
```
Comment (akramkazmi71) [inline src/App.js:21]: Dont forget to change the ICON too. 

You can create a new icon too.

**Review by akramkazmi71 — CHANGES_REQUESTED** [review]

**Pull request merged**